src/systemTest/java/randoop/main/RandoopSystemTest.java

```diff
@@ -1063,7 +1064,7 @@ public void runFlakyTest() {
             "flaky.FlakyClass.multiply(int, int) include");
 
     List<String> expectedFlakyMethodsInOrder = new ArrayList<>();
-    expectedFlakyMethodsInOrder.add("flaky.FlakyClass.flakyDefaultHashCode()");
+    expectedFlakyMethodsInOrder.add("flaky.FlakyClass.flakyDefaultHashCode(flaky.FlakyClass)");
```

Comment:
This change doesn't seem right to me.
The method `flakyDefaultHashCode` takes no arguments.  Nowhere else does Randoop require explicitly giving the receiver type.  I don't think that the receiver type should be listed in this output.